cloudformation/mongo-opsmanager-backup.template

```diff
@@ -16,6 +16,25 @@
         "PROD"
       ]
     },
+    "Stack": {
+      "Description": "Stack name",
+      "Type": "String"
+    },
+    "SshKeyFileS3Url": {
+      "Description": "S3 URL for a list of public SSH keys to use for the default ubuntu user",
+      "Type": "String",
+      "Default": ""
+    },
+    "EBSOptions": {
+      "Description": "Extra parameters to add-encrypted script",
+      "Type": "String",
+      "Default": "-t io1 -i 1000"
```

Comment:
Yeah, we had some issues with the provisioned IOPS originally and 1000 was the lowest value we tried that didn't lead to a painfully slow initial sync.

We're using magnetic disk without provisioned IOPS for CODE now, but I'd love to hear your thoughts/recommendations around how much to provision for SSD in PROD.